data/meson.build

```diff
@@ -0,0 +1,40 @@
+# Setup configuration data for .desktop entries
+desktop_conf = configuration_data()
+desktop_conf.set('bindir', join_paths(prefix, bindir))
+desktop_conf.set('datadir', join_paths(prefix,datadir))
```

Comment:
nitpick: space after the comma

I would also use `/`, but you can keep this if you want.